media-video/vdr/vdr-2.4.6.ebuild

```diff
@@ -0,0 +1,184 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic multilib optfeature toolchain-funcs user-info
+
+DESCRIPTION="Video Disk Recorder - turns a pc into a powerful set top box for DVB"
+HOMEPAGE="http://www.tvdr.de/"
+SRC_URI="ftp://ftp.tvdr.de/vdr/${P}.tar.bz2
+	mainmenuhooks? ( http://vdr.websitec.de/download/${PN}/vdr-2.4.1/vdr-2.4.1_mainmenuhook-1.0.1.patch.bz2 )
+	menuorg? ( https://projects.vdr-developer.org/projects/plg-menuorg/repository/revisions/master/raw/vdr-patch/vdr-menuorg-2.3.x.diff )
+	naludump? ( http://www.udo-richter.de/vdr/files/vdr-2.1.5-naludump-0.1.diff )
+	ttxtsubs? ( http://vdr.websitec.de/download/${PN}/vdr-2.4.4/vdr-2.4.4_ttxtsubs.patch.bz2 )"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+IUSE="bidi debug demoplugins html keyboard mainmenuhooks menuorg naludump pinplugin permashift systemd ttxtsubs verbose"
+
+COMMON_DEPEND="
+	>=media-libs/fontconfig-2.4.2
+	>=media-libs/freetype-2
+	sys-libs/libcap
+	virtual/jpeg:*
```

Comment:
I see there's `fowners` call in `src_install`, so you need to add the proper acct-* packages here.

Reply:
see previous comment about acct-* packages